testing/src/main/java/org/oppia/android/testing/profile/ProfileTestHelper.kt

```diff
@@ -13,7 +14,13 @@ class ProfileTestHelper @Inject constructor(
   private val profileManagementController: ProfileManagementController,
   private val testCoroutineDispatchers: TestCoroutineDispatchers
 ) {
-  /** Creates one admin profile and one user profile. Logs in to admin profile. */
+
+  private val observer = Observer<AsyncResult<Any?>> { }
+
+  /** Creates one admin profile and one user profile. Logs in to admin profile.
```

Comment:
Please fix KDoc formatting to follow style guide & be consistent with other KDocs in the codebase (there are multiple formatting issues here). Ditto below.

Reply:
got it, done. https://kotlinlang.org/docs/reference/kotlin-doc.html